src/styles/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  display: flex;
+  &__list {
+    display: flex;
+    flex-direction: row;
```

Comment:
default value, remove it

Reply:
<img width="316" alt="Снимок экрана 2022-12-15 в 07 16 35" src="https://user-images.githubusercontent.com/102182808/207778396-5d538011-6bd6-48ba-9e73-0f3fa432a68e.png">
Yes, it is. Because you can't add `flex direction` to your `reset.css` file